handlers/main.yml

```diff
@@ -2,6 +2,9 @@
 
 - name: restart nginx
   become: yes
-  service:
-    name: nginx
+  systemd:
```

Comment:
Why the switch to `systemd` here? `service` should already handle `systemd` setups.

Reply:
Cleanup!

https://github.com/ergontech-ansible-roles/nginx/pull/1/files#diff-2444ad0870f91f17ca6c2a5e96b26823L10

and - https://askubuntu.com/questions/911525/difference-between-systemctl-init-d-and-service
>If you have the option to use systemctl for your service control I recommend using that one.

Reply:
I'm still confused since Ansible's `service` module says it supports `systemd` systems. If the `service` module does work with systemd configurations, this code change would make our code less compatible with other systems, but that's not a big concern at the moment.

Reply:
I was just cleaning up Colby's TODO 😭